src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -495,7 +495,9 @@ export function transform(formConfig, form) {
             .join(),
           underDoctorHopitalCarePast12M:
             unemployability.underDoctorsCare || unemployability.hospitalized,
-          mostEarningsInAYear: unemployability.mostEarningsInAYear.toString(),
+          mostEarningsInAYear: (
+            unemployability.mostEarningsInAYear || ''
+          ).toString(),
```

Comment:
Turns out there's a bug in the form app tester where field names ending in "Year" are treated like date inputs or something (still need to figure out the exact problem). This meant that `mostEarningsInAYear` didn't get entered, and it caused an error here because the we were calling `toString` on `undefined`.